setup.py

```diff
@@ -44,6 +39,7 @@
     python_requires=">=3.7",
     packages=["aestream"],
     install_requires=["numpy"],
+    extras_require={"torch": ["torch"]},
```

Comment:
Same as above

Reply:
But is this a problem? Can't we just add future dependencies as we want? As far as I understand, the option to enable torch (`pip install aestream[torch]`) is independent of other extras.